av_novatel_launch/launch/av_novatel.launch.xml

```diff
@@ -1,6 +1,6 @@
 <launch>
   <group>
-    <push_ros_namespace namespace="/sensor/gps/rear"/>
+    <push_ros_namespace namespace="/sensor/gps"/>
```

Comment:
Why was this changed? At least should be written in the commit msg, if not the changelog/PR description

Reply:
Understood, perhaps this was to differentiate it with the mysterious garmin gps